collect_app/src/main/java/org/odk/collect/android/formlists/FormListSortingDialog.kt

```diff
@@ -0,0 +1,37 @@
+package org.odk.collect.android.formlists
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import org.odk.collect.android.R
+import org.odk.collect.android.adapters.SortDialogAdapter
+
+class FormListSortingDialog(
```

Comment:
Let's name this `FormListSortingBottomSheetDialog` - for a second there, I thought you'd changed the UI to a normal dialog.

Reply:
> I thought you'd changed the UI to a normal dialog.

Why? no it's still a bottom dialog.

Reply:
Because the name didn't mention "BottomSheet" and just said "Dialog". I think it's good to call out that this isn't a "modal" dialog (which would be my assumption from the name).